packages/ui/src/DetailInfo/DetailInfo.tsx

```diff
@@ -1,66 +1,99 @@
 import { ReactNode } from 'react'
 import { styled } from 'styled-components'
+import Divider from '@mui/material/Divider'
+import { useReleaseChannel } from '@ui-kit/hooks/useLocalStorage'
+import ActionInfo, { ActionInfoProps } from '@ui-kit/shared/ui/ActionInfo'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { ReleaseChannel } from '@ui-kit/utils'
 import Box from 'ui/src/Box/Box'
 import Loader from 'ui/src/Loader/Loader'
 
+const { Spacing } = SizesAndSpaces
+
 type Variant = 'error' | 'warning' | 'success' | ''
-type Size = 'xs' | 'sm' | 'md' | 'lg'
+
+/**
+ * Maps the `variant` prop to the corresponding `valueColor` prop for the `ActionInfo` component.
+ * This should be removed when we get rid of this file and use `ActionInfo` directly.
+ * */
+const VariantToColorMap = {
+  '': 'textPrimary',
+  error: 'error',
+  warning: 'warning',
+  success: 'success',
+} satisfies Record<Variant, ActionInfoProps['valueColor']>
 
 type Props = {
   children: ReactNode
-  action?: ReactNode
   className?: string
   isBold?: boolean | null
   isDivider?: boolean
   isMultiLine?: boolean
   label?: ReactNode
   loading?: boolean
   loadingSkeleton?: [number, number]
-  size?: Size
-  textLeft?: boolean
   tooltip?: ReactNode
   variant?: Variant
 }
 
-const DetailInfo = ({
-  action: Action,
+const OldDetailInfo = ({
   className,
   isBold,
   isDivider,
   label,
   loading,
   loadingSkeleton,
-  tooltip: Tooltip,
+  tooltip,
   variant,
   children,
-  size = 'sm',
-  ...props
-}: Props) => {
-  const classNames = `${className} ${isDivider ? 'divider' : ''}`
-
-  return (
-    <Wrapper
-      {...props}
-      size={size}
-      className={classNames}
-      grid
-      gridAutoFlow="column"
-      gridColumnGap={2}
-      isDivider={isDivider}
-      fillWidth
-    >
-      {label && <DetailLabel>{label}</DetailLabel>}
-      <DetailValue haveLabel={!!label} isBold={isBold} variant={variant}>
-        {loading && <Loader skeleton={loadingSkeleton} />}
-        {!loading && (
-          <>
-            {children || '-'} {!!Tooltip && Tooltip} {Action && Action}
-          </>
-        )}
-      </DetailValue>
-    </Wrapper>
-  )
-}
+}: Props) => (
+  <Wrapper
+    className={`${className} ${isDivider ? 'divider' : ''}`}
+    grid
+    gridAutoFlow="column"
+    gridColumnGap={2}
+    isDivider={isDivider}
+    fillWidth
+  >
+    {label && <DetailLabel>{label}</DetailLabel>}
+    <DetailValue haveLabel={!!label} isBold={isBold} variant={variant}>
+      {loading && <Loader skeleton={loadingSkeleton} />}
+      {!loading && (
+        <>
+          {children || '-'} {!!tooltip && tooltip}
+        </>
+      )}
+    </DetailValue>
+  </Wrapper>
+)
+
+const NewDetailInfo = ({
+  className,
```

Comment:
Is this something we want to keep, or just temporary because we're migrating? Imo we should add a todo once this feature goes out of beta so only the advised `sx` becomes available.

Reply:
It's used here only but I'm not sure how to deal with it :thinking: I don't see matching css rules
https://github.com/curvefi/curve-frontend/blob/3898365f0c760726669e5d4bc9c3d4fc34b84b7a/apps/main/src/dex/components/PagePool/components/DetailInfoEstLpTokens.tsx#L37

Reply:
Prob can be removed